src/components/AddOffering/AddItems/ItemImagesPreview.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
```

Comment:
This file is a complete duplication of the `ImageCarousel.js` file in /Item folder of `postings-integration branch`. This file should be removed when that branch is integrated.